src/systems/audio-system.js

```diff
@@ -137,11 +137,21 @@ export class AudioSystem {
     this.outboundAnalyser.connect(this.mediaStreamDestinationNode);
     this.audioContextNeedsToBeResumed = false;
 
+    this.mediaNodes = [];
```

Comment:
I see where we are adding and removing to this, but I don't see anything that actually reads them. Maybe we don't need to keep track of them?

Also to remove duplicate code with `addAudioToX` and `removeAudioFromX` we could just have a named set of tracks like:

mediaNodesMight be nice to just use named "tracks"
```js
this.mixer = {
   avatar:  this.audioContext.createGain(),
   media:  this.audioContext.createGain()
}
```
And then passing that as a paramter:
```js
addAudio(mixerTrack, audioNode) {
```

Reply:
Thanks for that suggestion, it makes everything much clearer.